xrviz/example.py

```diff
@@ -0,0 +1,29 @@
+from .dashboard import Dashboard
+
+
+def example(show=True):
+    """
+    Generates interface for the sample dataset.
+
+    Parameters
+    ----------
+    show: bool (True)
+        Whether to directly execute the interface. If True, a new browser tab
+        will be opened, and the function will block until interrupted. If
+        False, the Dashboard instance is returned without being executed.
+    """
+    from .sample_data import great_lakes
+
+    great_lakes = great_lakes.set_coords(['lat', 'lon'])
+    initial_params = {'projection_ops': {'is_geo': True,
```

Comment:
I have added a way to initialise projection panel values by passing values as `dict`. As of now we can only  initialise projections panel. Please let me know if this is correct approach before I add this method in other panels as well.

Reply:
I would prefer a name here which is identical to the tab title, "projection", in this case.
There is also an argument for having this flat, single-level, rather than nested, since no widget names are duplicated anywhere in the interface. Each panel can use only the keys which are relevant to its widgets.